### PR TITLE
Site specific vars for staging, wayland and berwyn

### PR DESCRIPTION
**Pull request by jhackettpps**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -48,16 +48,16 @@ services:
...
-                - XDEBUG_IP=127.0.0.1 # Replace with your machines IP: $ ipconfig getifaddr en0 
+                 - HUB_DB_ENV_MYSQL_DATABASE=hubdb
+                 - HUB_DB_ENV_MYSQL_USER=hubdb_user
+                 - HUB_DB_ENV_MYSQL_PASSWORD
+                 - HUB_DB_PORT_3306_TCP_ADDR=hub-db
+                 - HUB_EXT_FILE_URL=http://localhost:8181/sites/default/files
+                 - PHP_MEMORY_LIMIT=500M
+                 - PHP_UPLOAD_MAX_FILE_SIZE=500M
+                 - PHP_POST_MAX_SIZE=500M
+                 - SIMPLETEST_DB=mysql://hubdb_user:hubdb_pass@localhost/hubdb
```
Comment (Tiki-God) [inline docker-compose.yml]: I can't find the SIMPLETEST_DB var in prod, is this needed for testing or is it just a legacy variable that is no longer used?

File: sites/docker-compose.berwyn.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+    hub-be:
+        environment:
+            DRUPAL_APP_URI: https://drupal.digital-hub-stage.hmpps.dsd.io
```
Comment (Tiki-God) [inline sites/docker-compose.berwyn.yml]: DRUPAL_APP_URI only seems to apply to hub-node so shouldn't be needed for the hub-be container.

File: docker-compose.yml [hunk 3]
```diff
@@ -48,16 +48,16 @@ services:
...
-                - PHP_UPLOAD_MAX_FILE_SIZE=500M
-                - PHP_POST_MAX_SIZE=500M
-                - SIMPLETEST_DB=mysql://hubdb_user:hubdb_pass@localhost/hubdb
-                - XDEBUG_IP=127.0.0.1 # Replace with your machines IP: $ ipconfig getifaddr en0 
+                 - HUB_DB_ENV_MYSQL_DATABASE=hubdb
+                 - HUB_DB_ENV_MYSQL_USER=hubdb_user
+                 - HUB_DB_ENV_MYSQL_PASSWORD
+                 - HUB_DB_PORT_3306_TCP_ADDR=hub-db
+                 - HUB_EXT_FILE_URL=http://localhost:8181/sites/default/files
+                 - PHP_MEMORY_LIMIT=500M
```
Comment (Tiki-God) [inline docker-compose.yml]: I'd like the PHP_MEMORY_LIMIT to match what is in production to catch memory issues quicker.  Both prod sites use 256M

File: sites/docker-compose.berwyn.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            - /content/moj_dhub_prod001_app/usr/share/nginx/html/moj_be/sites/default/files:/var/www/html/sites/default/files
+
+    hub-db:
+        volumes:
+            - /data/hub-db:/var/lib/mysql:rw
+
+    hub-matomo:
+        volumes:
+            - /data/moj_dhub_matomo_config:/var/www/html/config:rw
+
```
Comment (Tiki-God) [inline sites/docker-compose.berwyn.yml]: hub-matomo-db will need the data volumes adding

/data/moj_dhub_matomo_db/var/lib/mysql/:/var/lib/mysql/

File: sites/docker-compose.berwyn.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    hub-be:
+        environment:
+            DRUPAL_APP_URI: https://drupal.digital-hub-stage.hmpps.dsd.io
+            HUB_DB_ENV_MYSQL_DATABASE: hubdb
+            HUB_DB_ENV_MYSQL_USER: hubdb_user_2y478gt
+            HUB_DB_PORT_3306_TCP_ADDR: hub-db
+            HUB_EXT_FILE_URL: http://digital-hub.bwi.dpn.gov.uk:11001/sites/default/files
+            PHP_MEMORY_LIMIT: 256M
+            PHP_UPLOAD_MAX_FILE_SIZE: 1024M
+            PHP_POST_MAX_SIZE: 1024M
```
Comment (Tiki-God) [inline sites/docker-compose.berwyn.yml]: Should we reduce the UPLOAD and POST max sizes to 500M to match staging?  Only staging has any content uploads done.  While not really an issue, I feel warm and fuzzy with the consistency.

File: sites/docker-compose.berwyn.yml (hunk 4 above)
Comment (Tiki-God) [reply sites/docker-compose.berwyn.yml]: Same for Wayland, and maybe staging?

File: sites/docker-compose.staging.yml [hunk 6]
```diff
@@ -15,3 +15,7 @@ services:
     hub-matomo:
         volumes:
             - /data/moj_dhub_matomo_config:/var/www/html/config:rw
+
+        hub-node:
```
Comment (Tiki-God) [inline sites/docker-compose.staging.yml]: Indentation

File: sites/docker-compose.wayland.yml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+version: "3"
+
+services:
+    hub-be:
+        environment:
```
Comment (Tiki-God) [inline sites/docker-compose.wayland.yml]: Seems to be missing a lot of environment variables for database connectivity and such

File: docker-compose.yml (hunk 1 above)
Comment (jhackettpps) [reply docker-compose.yml]: Honestly, no idea

**Pull request merged**